flowkit/cmd/flow-schema/flow-schema.go

```diff
@@ -54,11 +55,15 @@ func main() {
 			os.Exit(1)
 		}
 
-		if string(fileContents) != string(json) {
+		if normalizeNewlines(string(fileContents)) != normalizeNewlines(string(json)) {
```

Comment:
JSON schema generator checks for changes, we need to normalize CRLF to LF otherwise Windows machine will show changes compared to UNIX-generated schema.